### ISSUE TITLE
Strange behavior after updating to 2.10 (and latest as well)

### ISSUE THREAD

**Issue by FHeNuS** [exeldro/obs-move-transition#204]
I've updated to the latest version 2.12 and without any other change my transitions now do not seem to work as they were. I have different filters that change the shape of a source while the transition is happening, and while on the 2.9.8 the bug would only appear if I cutted the transition midway and then came back, on 2.10 and 2.12 the bug is consistent. The first part of the video is the bug, the second is how it should look like (with 2.9.8 version). The filters popup is to show that the filters are being "applied" at the beggining of the transition, but something seemed to have changed on the internal processing.

https://github.com/exeldro/obs-move-transition/assets/288856/bccba5f5-39df-4e15-93e8-befbc36bb72c

**Comment by exeldro** [comment]
It try to replicate the issue on my system, but have not succeeded yet. From the video it looks like move source filters on a nested scene triggered on scene transition. Can you provide a scene collection with the issue or explain what move settings I need to use to get the issue?

**Comment by FHeNuS** [comment]
Definitely! Will create one later today and provide you with the files.
